### DISCUSSION TITLE
NES Multicart

### DISCUSSION THREAD

**Discussion by VamPikmin** [sanni/cartreader#1050] [Need help]
Hi there,

I have this childhood Nes multicart- Super HIK 4 in 1

It contains Felix the Cat, Mario 5, Jurassic Park and J League Soccer

From what I can see all are Mapper 4 128 kb PRG and CHR

When I insert it into the cart reader it gets detected as Super Mario Bros 2 and I can dump it but how do I get the other 3 games?

![SuperMarioBros2-0](https://github.com/user-attachments/assets/e6b44735-7b16-4122-b24d-91d2e90d840b)
![SuperMarioBros2-1](https://github.com/user-attachments/assets/28655b68-8720-4ecd-b392-b763621fe296)

**Comment by nsx0r** [comment]
Hello,
Unfortunately, many multicarts get detected by the OSCR as one of the games it contains, so you'll have to manually select the mapper and the ROM size.
I don't remember seeing this exact menu screenshot so I can't tell you for sure which mapper it is, my guess is 315 or 366 so I added both to the code just now.
If those don't work, I will need more info like cartridge label and PCB pictures but the best way to identify a mapper is analyzing a mapper 0 dump, so message me on Discord if that's the case (don't post ROM links on github please, even broken/partial ones).

**Reply by VamPikmin** [reply]
Hello nsx0r,
Thank you. 

Sorry I'm still new to this, once I select Super Mario Bros 2 and change the mapper 315 or 366 do I need to change the PRG and CHR to something bigger than 128k?

![Super HIK 4 in 1](https://github.com/user-attachments/assets/a8bb7d9c-7c24-4459-ac59-20112ca1b3af)

**Reply by nsx0r** [reply]
yes, PRG = CHR = 512
it could also be mapper 52 which I just corrected

**Reply by VamPikmin** [reply]
I've tried that with 315 and 366 before I went to bed, when set to 512 and tried to load the file in the emulator it doesn't work just get a three tone colour, when I select Mario I get corrupted graphics.

This morning I tried mapper 52 and same.

So the next step is to dump it with mapper 0 and 512k?
Thanks

**Reply by nsx0r** [reply]
mapper 0 with PRG=32 and CHR=8

**Reply by VamPikmin** [reply]
Resolved by nsx0r

Thank you!
